### PR TITLE
List user

### PR DESCRIPTION
**Pull request by at-cuclt**
@at-dungnh please help me review and merge this, thanks.

### CODE REVIEW

File: resources/views/admin/users/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@section('content')
+<!-- Page Heading -->
+<div class="row">
+    <div class="col-lg-12">
+        <h1 class="page-header">
+            {!! trans('user.manage_user') !!}
+        </h1>
+        <ol class="breadcrumb">
+            <li>
+                <i class="fa fa-dashboard"></i>  <a href="index.html">Dashboard</a>
```
Comment (dzungnguyenh) [inline resources/views/admin/users/index.blade.php]: [rank A] please use this text in language file

File: resources/views/admin/layouts/partials/_javascript.blade.php [hunk 2]
```diff
@@ -1,6 +1,4 @@
 	<!-- COMBINE BOOTSTRAP AND JQUERY JS -->
     <script src="backend/js/vendor.js"></script>
-    <script type="text/javascript" src="backend/js/myScript.js">
-           
-    </script>
+    <script type="text/javascript" src="backend/js/myScript.js"></script>
```
Comment (dzungnguyenh) [inline resources/views/admin/layouts/partials/_javascript.blade.php]: [rank A] "myScript.js" ??? what does it mean?

File: resources/views/admin/layouts/partials/_sidebar.blade.php [hunk 3]
```diff
@@ -7,7 +7,7 @@
                         <a href="#"><i class="fa fa-fw fa-edit"></i> {!! trans('labels.createaccount') !!}</a>
                     </li>
                     <li>
-                        <a href="#"><i class="fa fa-fw fa-table"></i> {!! trans('labels.ManageUser') !!}</a>
+                        <a href="{!! URL('/user') !!}"><i class="fa fa-fw fa-table"></i> {!! trans('labels.ManageUser') !!}</a>
```
Comment (dzungnguyenh) [inline resources/views/admin/layouts/partials/_sidebar.blade.php]: [rank A] please use regular form for this function

File: resources/views/admin/users/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+@section('content')
+<!-- Page Heading -->
+<div class="row">
+    <div class="col-lg-12">
+        <h1 class="page-header">
+            {!! trans('user.manage_user') !!}
+        </h1>
+        <ol class="breadcrumb">
+            <li>
+                <i class="fa fa-dashboard"></i>  <a href="{!! URL('/') !!}">{!! trans('labels.Dashboard') !!}</a>
```
Comment (dzungnguyenh) [inline resources/views/admin/users/index.blade.php]: [rank A] please use regular form for this function and language key is in snake style

File: resources/views/admin/users/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+            </li>
+        </ol>
+    </div>
+</div>
+<!-- /.row -->
+
+<div class="row">
+    <div class="col col-md-12">
+        <div class="table-responsive">
+            <table id="myTable" class="text-center display" cellspacing="0" width="100%">
```
Comment (dzungnguyenh) [inline resources/views/admin/users/index.blade.php]: [rank A] please declare element id clearly more. "myTable" is not meaning

File: resources/views/admin/users/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                        <th class="text-center">{!! trans('user.full_name') !!}</th>
+                        <th class="text-center">{!! trans('user.birthday') !!}</th>
+                        <th class="text-center">{!! trans('user.gender') !!}</th>
+                        <th class="text-center">{!! trans('user.phone') !!}</th>
+                        <th class="text-center">{!! trans('user.address') !!}</th>
+                        <th class="text-center">{!! trans('user.more') !!}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <?php $i=1 ?>
```
Comment (dzungnguyenh) [inline resources/views/admin/users/index.blade.php]: [rank A] I think it should be $order_number or $index. It's better to readable

File: resources/views/admin/layouts/partials/_javascript.blade.php (hunk 2 above)
Comment (at-cuclt) [inline resources/views/admin/layouts/partials/_javascript.blade.php]: That js for dataTable. I will rename it.

File: resources/lang/en/user.php [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
+
+    'no' => 'NO',
```
Comment (dzungnguyenh) [inline resources/lang/en/user.php]: [rank A] 'NO' should be 'No.'

File: app/Http/Middleware/Authenticate.php [hunk 8]
```diff
@@ -22,7 +22,11 @@ public function handle($request, Closure $next, $guard = null)
             if ($request->ajax() || $request->wantsJson()) {
                 return response('Unauthorized.', 401);
```
Comment (dzungnguyenh) [inline app/Http/Middleware/Authenticate.php]: [rank B] what does "401" means? Please refer to other about this. I did raise this issue before.

File: public/backend/js/dataTable.js [hunk 9]
```diff
@@ -0,0 +1,4 @@
+$(document).ready(function(){
+  var frmId = $('table').attr('id');
```
Comment (dzungnguyenh) [inline public/backend/js/dataTable.js]: [rank C] why do you update this code like that? What's happen if we have many table elements in our page? If you cannot understand my opinions, please tell me to clarify.

File: resources/views/admin/auth/login.blade.php [hunk 10]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <title>Login Admin</title>
```
Comment (dzungnguyenh) [inline resources/views/admin/auth/login.blade.php]: [rank A] please put this to language file

File: resources/views/admin/users/index.blade.php [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+                        <th class="text-center">{!! trans('user.full_name') !!}</th>
+                        <th class="text-center">{!! trans('user.birthday') !!}</th>
+                        <th class="text-center">{!! trans('user.gender') !!}</th>
+                        <th class="text-center">{!! trans('user.phone') !!}</th>
+                        <th class="text-center">{!! trans('user.address') !!}</th>
+                        <th class="text-center">{!! trans('user.more') !!}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <?php $index=1 ?>
```
Comment (dzungnguyenh) [inline resources/views/admin/users/index.blade.php]: [rank B] where is this semicolon?

File: public/backend/js/dataTable.js [hunk 12]
```diff
@@ -0,0 +1,4 @@
+$(document).ready(function(){
```
Comment (dzungnguyenh) [inline public/backend/js/dataTable.js]: [rank A] will this file using for all of table list in your project? Will you put other js code in this file in future?

**Comment by dzungnguyenh** [comment]
Merged! [LGTM](http://lgtm.in/i/e3sGUcOdq)

**Pull request merged**